rust-launcher/src/property.rs

```diff
@@ -85,7 +85,7 @@ fn check_file_for_property(file: File, key: &str) -> Option<String> {
                     None => {}
                     Some(kvv) => {
                         if kvv.key.eq(key) {
-                            return Some(escape_unicode(kvv.value));
+                            return Some(escape_unicode(kvv.value.replace(r"\:", ":")));
```

Comment:
Can you please explain why it is necessary?
Will it not be soved by https://github.com/rust-lang/rust/issues/27791  ? Which should be heading to stable soon.

Reply:
The colon is a delimiter for Java properties. Therefore it is masked during serialization. When used in Java getProperty, it will automatically unmask. In Rust you have to take care of the unmasking yourself. How it should help escape_unicode (), does not open to me.

Reply:
Colon xor equals is delimiter, depends what comes first. Isnt that?

Reply:
ping?

Reply:
The following line has itweb-settings written in my deplyments.properties.

deployment.jre.dir=C\:\\Program Files\\AdoptOpenJDK\\jdk-8.0.202.08\\jre

Here is the equal sign the delimiter, but java has masked the colon anyway. Without the replace, this is an invalid path, which will simply not be accepted.

Reply:
The following line has itweb-settings written in my deployments.properties.

deployment.jre.dir=C\:\\Program Files\\AdoptOpenJDK\\jdk-8.0.202.08\\jre

Here is the equal sign the delimiter, but java has masked the colon anyway. Without the replace, this is an invalid path, which will simply not be accepted.

Reply:
The following line has itweb-settings written in my deployments.properties.

`deployment.jre.dir=C\:\\Program` `Files\\AdoptOpenJDK\\jdk-8.0.202.08\\jre`

Here is the equal sign the delimiter, but java has masked the colon anyway. Without the replace, this is an invalid path, which will simply not be accepted.

Reply:
Ok. gave sense now.